test/test-framework-spec.md

```diff
@@ -66,6 +70,24 @@ Ballerina tests are defined using a set of annotations. The following are the an
    <td>Function specified will be run once before any of the tests in the test suite is run.
    </td>
   </tr>
+  <tr>
+   <td>@test:BeforeGroups {}
+   </td>
+    <td>Function specified will be run before any of the tests belonging to the specified groups are executed.
+<p>
+Annotation value fields :
+<ul>
+
+<li>groups : [“&lt;test group name&gt;”, ...] 
+<ul>
+
+<li>List of group names that this function should run before
+</li> 
+</ul>
+</li> 
+</ul>
+   </td>
+  </tr>
```

Comment:
We should have `serialExecution` flag for the @test:Config annotation now.

Reply:
Shall we add after the U9 release?